### PR TITLE
Add docs for database schema migration

### LINKED ISSUE #239: Add docs on how to upgrade Task schema
For users who may have already have an existing DB based on the other schema

### PR DESCRIPTION
**Pull request by markpollack**
Fixes #239

### CODE REVIEW

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/cf-tasks.adoc [hunk 1]
```diff
@@ -25,6 +25,17 @@ It's also important to note that there is no Apps Manager support for tasks as o
 release. When running applications as tasks through Spring Cloud Data Flow, the only way
 to view them within the context of CF is via the plugin mentioned above.
 
+== Task Database Schema
+
+The database schema for Task applications was changed slighlty from the 1.0.x to 1.1.x version of
+Spring Cloud Task.  Since Spring Cloud Data Flow automatically creates the database schema if it is
+not present upon server startup, you may need to update the schema if you ran a 1.0.x version of the
+Data Flow server and now are upgrading to the 1.1.x version.  You can find the migration scripts
+link:https://github.com/spring-cloud/spring-cloud-task/commits/master/spring-cloud-task-core/src/main/resources/org/springframework/cloud/task/migration[here]
```
Comment (cppwfs) [inline spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/cf-tasks.adoc]: Probably want to use this link instead.  https://github.com/spring-cloud/spring-cloud-task/tree/1.1.0.RELEASE/spring-cloud-task-core/src/main/resources/org/springframework/cloud/task/migration.  Can change on merge if you agree.

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/cf-tasks.adoc [hunk 2]
```diff
@@ -25,6 +25,17 @@ It's also important to note that there is no Apps Manager support for tasks as o
...
+in the Spring Cloud Task repository.  The documentation for link:https://docs.cloudfoundry.org/devguide/deploy-apps/ssh-services.html[Accessing Services with Diego SSH]
```
Comment (cppwfs) [inline spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/cf-tasks.adoc]: Spring Cloud Task `Github` repository.

Comment (cppwfs) [inline spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/cf-tasks.adoc]: will update if you agree.

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/cf-tasks.adoc (hunk 1 above)
Comment (markpollack) [inline spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/cf-tasks.adoc]: ok

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/cf-tasks.adoc (hunk 2 above)
Comment (markpollack) [inline spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/cf-tasks.adoc]: ok

**Comment by cppwfs** [comment]
LGTM

**Comment by cppwfs** [comment]
Fixes Applied, Squashed Merged.

**Pull request closed without merging**